shell/common/gin_helper/callback.cc

```diff
@@ -15,7 +15,7 @@ namespace {
 struct TranslaterHolder {
   explicit TranslaterHolder(v8::Isolate* isolate)
       : handle(isolate, v8::External::New(isolate, this)) {
-    handle.SetWeak(this, &GC, v8::WeakCallbackType::kFinalizer);
+    handle.SetWeak(this, &GC, v8::WeakCallbackType::kParameter);
```

Comment:
hooooold up. this doesn't seem correct??

Reply:
Oop, I'm pretty sure this was me, here's the CL that changed it: https://chromium-review.googlesource.com/c/v8/v8/+/3596174 It looks like `kFinalizer` is now deprecated and removed, but `kParameter` may not be the right thing to replace it with

Reply:
On closer inspection, it might be right after all!

```
  /**
   * Passes a user-defined void* parameter back to the callback.
   */
  kParameter,
```

vs the now-removed

```
  /**
   * Passes a user-defined void* parameter back to the callback. Will do so
   * before the object is actually reclaimed, allowing it to be resurrected. In
   * this case it is not possible to set a second-pass callback.
   */
  kFinalizer V8_ENUM_DEPRECATED("Resurrecting finalizers are deprecated "
                                "and will not be supported going forward.")
```

in our case we just call `delete` on the parameter, which I don't think can resurrect the object, so we're probably fine to use `kParameter`. cc @zcbenz in case this analysis is incorrect.